### PR TITLE
Add built-in modules list

### PR DESCRIPTION
**Pull request by Nokitoo**
List all the built-in modules in the userguide.

### CODE REVIEW

File: docs-sources/includes/_Modules.md [hunk 1]
```diff
@@ -3,17 +3,19 @@
 ## External helper modules
 
 Some mage modules are optional but simplify the use of the library.
+Unlike other modules, they don't expose usercommands, they just act as helpers.
```
Comment (stelcheck) [inline docs-sources/includes/_Modules.md]: I think that is incorrect, e.g. archivist.

Comment (Nokitoo) [reply docs-sources/includes/_Modules.md]: Is there an archivist external module ?

Comment (stelcheck) [reply docs-sources/includes/_Modules.md]: ah sorry, I misread that.

Comment (Nokitoo) [reply docs-sources/includes/_Modules.md]: I will rephrase so there is no mistake

**Pull request merged**